libvast/src/system/spawn_sink.cpp

```diff
@@ -82,7 +82,8 @@ maybe_actor spawn_zeek_sink(caf::local_actor* self, spawn_arguments& args) {
   if (!args.empty())
     return unexpected_arguments(args);
   auto writer = std::make_unique<format::zeek::writer>(
-    get_or(args.inv.options, category + ".write", defaults_t::write));
+    get_or(args.inv.options, category + ".write", defaults_t::write),
+    !caf::get_or(args.inv.options, category + ".no-timestamp-tags", false));
```

Comment:
`s/no-timestamp-tags/show-timestamp-tags/`?

Otherwise we're dealing with a triple negative here :D

Reply:
I thought about that, but that makes only sense if it is disabled by default, because then the flag `--show-timestamp-tags` must be provided to achieve the normal Zeek output. I'd rather disable it selectively.

Not sure how to avoid the triple negation. 😜

Reply:
Not sure that I follow, you could have `--show-timestamp-flags=true` be the default and disable them with `--show-timestamp-flags=false` in the integration tests?

Reply:
What I meant is that, for a boolean flag, the expectation is usually toggle it without setting it with `--flag=false` or `--flat=true`. Usually it's just `--flag` and the absence of `--flag` implies the opposite. This is why I named the flag `--no-X` instead of `--show-X`. Does that make sense?

I you prefer to avoid these negations, I'm happy to flip things on its head with the changes you proposed.

Reply:
The `--no-X` flag makes sense in isolation, but it doesn't fit with the rest of our command-line syntax which uses `--enable-X/--disable-X` for these things (`--disable-community-id`, `--enable-metrics`, ...)

But I don't have a too strong opinion on this, if you like the current `--no-timestamps-tag` feel free to just resolve the discussion and merge as-is.

Reply:
Okay, I switched to `--disable-timestamp-tags` to fit in the naming scheme.